libbeat/processors/actions/rename_fields.go

```diff
@@ -0,0 +1,76 @@
+package actions
+
+import (
+	"fmt"
+
+	"github.com/elastic/beats/libbeat/beat"
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/libbeat/processors"
+)
+
+type renameFields struct {
+	Fields        map[string]string
+	ignoreMissing bool
+}
+
+type fieldTuple struct {
+	From string `config:"from"`
+	To   string `config:"to"`
+}
+
+func init() {
+	processors.RegisterPlugin("rename_fields",
+		configChecked(newRenameFields,
+			requireFields("fields")))
+}
+
+func newRenameFields(c *common.Config) (processors.Processor, error) {
+	config := struct {
+		Fields        map[string]string `config:"fields"`
+		IgnoreMissing bool              `config:"ignore_missing"`
+	}{}
+	err := c.Unpack(&config)
+	if err != nil {
+		return nil, fmt.Errorf("fail to unpack the rename_fields configuration: %s", err)
+	}
+
+	f := &renameFields{
+		Fields:        config.Fields,
+		ignoreMissing: config.IgnoreMissing,
+	}
+	return f, nil
+}
+
+func (f *renameFields) Run(event *beat.Event) (*beat.Event, error) {
+
```

Comment:
space?

Reply:
It's going to be fine as long as you don't tell @andrewkroh :-D